ledger/participant-integration-api/src/test/suite/scala/platform/indexer/ha/IndexerStabilityPostgresSpec.scala

```diff
@@ -8,4 +8,7 @@ import com.daml.testing.postgresql.PostgresAroundEach
 final class IndexerStabilityPostgresSpec extends IndexerStabilitySpec with PostgresAroundEach {
 
   override def jdbcUrl: String = postgresDatabase.url
+
+  override def lockIdSeed: Int =
+    1000 // it does not matter in case of Postgres: it is always a different instance
```

Comment:
A different instance of db for each test class?

Reply:
correct